### DISCUSSION TITLE
Cannot Download Pyenv

### DISCUSSION THREAD

**Discussion by EricYuan8307** [Homebrew/discussions#3783] [Everyday usage]
I tried to install pyenv by Homebrew but failed.
brew install pyenv

but the terminal said this:
Warning: No available formula with the name "pyenv".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

**Comment by SMillerDev** [comment]
What does `brew doctor` say?

**Reply by EricYuan8307** [reply]
Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/opt/homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /opt/anaconda3/bin/icu-config
  /opt/anaconda3/bin/krb5-config
  /opt/anaconda3/bin/freetype-config
  /opt/anaconda3/bin/xslt-config
  /opt/anaconda3/bin/libpng16-config
  /opt/anaconda3/bin/libpng-config
  /opt/anaconda3/bin/xml2-config
  /opt/anaconda3/bin/python3.9-config
  /opt/anaconda3/bin/python3-config
  /opt/anaconda3/bin/curl-config
  /opt/anaconda3/bin/ncursesw6-config

Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/core) checkout master

Warning: Broken symlinks were found. Remove them with `brew cleanup`:
  /opt/homebrew/etc/bash_completion.d/brew-services

**Reply by SMillerDev** [reply]
> Warning: Homebrew/homebrew-core was not tapped properly! Run:
rm -rf "/opt/homebrew/Library/Taps/homebrew/homebrew-core"
brew tap homebrew/core

Will fix it

**Reply by EricYuan8307** [reply]
Reinstalling the HomeBrew will solve this problem
